artifacts.go

```diff
@@ -637,5 +637,7 @@ func run() error {
 }
 
 func main() {
-	run()
+	if err := run(); err != nil {
+		os.Exit(-1)
```

Comment:
Should return a positive return code, negative return codes are not supported by POSIX AFAIK, and their use is discouraged. Negative numbers may be used to communicate other aspects, such as whether the process died from a signal, so we should stay away from that range.